.idea/misc.xml

```diff
@@ -3,4 +3,9 @@
   <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="Java" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
+  <component name="ProjectRunConfigurationManager">
+    <configuration default="true" type="JUnit" factoryName="JUnit">
+      <shortenClasspath name="NONE" />

```

Comment:
that should actually be argfile.

The change did not actually come originally from this PR but occurred when I rebased it to `main` branch.

Reply:
Also this file is missplaced. I removed it from `main`.
If you want to manage `misc.xml` it needs to be in `intellij/workspace/update/.idea/misc.xml` instead.